packages/dd-trace/test/config.spec.js

```diff
@@ -284,6 +285,7 @@ describe('Config', () => {
       { name: 'appsec.blockedTemplateJson', value: undefined, origin: 'default' },
       { name: 'appsec.eventTracking.enabled', value: true, origin: 'default' },
       { name: 'appsec.eventTracking.mode', value: 'safe', origin: 'default' },
+      { name: 'appsec.sca.enabled', value: undefined, origin: 'default' },
```

Comment:
isn't this supposed to be value: null then ? :thinking:

Reply:
and if i'm right, why is it not failing in the CI ?

Reply:
With this regex `(?<!assert)\.include\(($|.+,)` I found 3 similar cases in the whole repo, all in this file, line 235, 503, 761.
We need to fix this before merging the current PR. Would you agree with that but in a different PR ? :laughing: 
I think we can fix it by just adding an array around the objects, and changing to deep equal ? like done so in line 1401:
```javascript
    expect(updateConfig.getCall(1).args[0]).to.deep.equal([
      { name: 'sampleRate', value: 0, origin: 'remote_config' }
    ])
```